returnn/torch/engine.py

```diff
@@ -52,6 +57,11 @@ def train(self):
     start_epoch, _ = self.get_train_start_epoch_batch(self.config)
     final_epoch = self.config_get_final_epoch(self.config)
 
+    if start_epoch == 1:
+      self._model_state = None
+    else:
+      self._model_state = self._load_model_state_from_file(epoch=start_epoch - 1)
```

Comment:
Why keeping this in an attrib? I would load the params directly into the model, and only when this is really needed.